lib/iris/tests/test_merge.py

```diff
@@ -79,9 +79,7 @@ def custom_coord_callback(cube, field, filename):
         # Load slices, decorating a coord with custom attributes
         cubes = iris.load_raw(self._data_path, callback=custom_coord_callback)
         # Merge
-        merged = iris.cube.CubeList._extract_and_merge(
-            cubes, constraints=None, strict=False, merge_unique=False
-        )
+        merged = iris.cube.CubeList(cubes).merge()
```

Comment:
Well.... I think that the purpose of this test function is to show that identical attributes are preserved in a merge.  
So I don't understand why the cube-private function was used here, because this isn't supposed to be testing that code, but the general merge code.  Apart from this call, `_extract_and_merge` is _only_ used by the extract methods in the Cube class.

I'm presuming this was a hack, put in when a plain merge couldn't do this  -- though it now does.
I can't prove that, because it dates to the original GitHub commit 8 years ago !